notebooks/new-calibrationStarVisit.ipynb

```diff
@@ -67,7 +80,7 @@
     "    afwDisplay.setDefaultBackend(\"matplotlib\" if False else \"firefly\")\n",
     "    os.environ['FIREFLY_URL'] = \"http://firefly01.cp.lsst.org:8080/firefly/\"\n",
     "\n",
-    "    disp = afwDisplay.Display(1, name='RHL', reopenPlot=True)\n",
```

Comment:
Do we need to commit these changes?

Reply:
I'll put something generic as the name (e.g. calibrationStarVisit). Same in the snippet above regarding the network setup.